### DISCUSSION TITLE
I have loaded a DXF. Now Im sort of stuck on what to do next.

### DISCUSSION THREAD

**Discussion by jtsheedy** [haplokuon/netDxf#253] [Q&A] [answered]
I have gotten netDXF loaded and iintegrated into my program as far as loading a DXF. This works fine and I can see all of my entities and everything in the file. The question I have is In ordeto view the DXF on screen do I have to write a drawing routine that shows all of the elements or is there something written in the package that does this already? 
I am using skiaSharp for my graphics and standard picturebox for drawing.

**Comment by haplokuon** [comment, marked as answer]
netDxf do not provide any kind of functionality to display the contents of the a DXF. The way you display the entities on the screen depends of the graphics engine.
